flask/README.md

```diff
@@ -200,4 +200,13 @@ A sample query you can try if it works is:
 
 ## Authentication and Authorization
 
-## Database Migrations
+## Database Schema Migrations
+
+Occasionally it might be required to update the database schema. To this end we use the [peewee-moves](https://github.com/timster/peewee-moves) tool.
```

Comment:
Why not use the official repo? (Coleifer)

Reply:
coleifer is the maintainer of peewee itself. I didnt mention it since the section is about the peewee-moves tool.